js/testcall.js

```diff
@@ -48,10 +46,6 @@ function sdpMap(sdp) {
   return sdpLines.join("\r\n");
 }
 
-function doLog(msg) {
-  console.log(msg);
-}
-
```

Comment:
idea: could consider leaving this function and some of its invocations and then only having it log if a local storage item or url parameter is present ie. "logging=true"